src/PerfView/StackViewer/PerfDataGrid.xaml

```diff
@@ -58,120 +58,121 @@
                     </TextBlock>
                 </WPFToolKit:DataGridTemplateColumn.Header>
             </WPFToolKit:DataGridTemplateColumn>
+
             <WPFToolKit:DataGridTextColumn 
```

Comment:
:memo: The diff here is bad, but this was just a drag/drop reordering of the columns.